src/Microsoft.DotNet.Wpf/src/PresentationCore/MS/internal/SystemDrawingHelper.cs

```diff
@@ -1,108 +1,72 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
 
-//
-//
+#nullable enable
+
 // Description: Helper methods for code that uses types from System.Drawing.
-//
 
 using System.IO;
+using Windows.Win32.Graphics.Gdi;
 
 namespace MS.Internal
 {
     internal static class SystemDrawingHelper
     {
-        // return true if the data is a bitmap
-        internal static bool IsBitmap(object data)
-        {
-            SystemDrawingExtensionMethods extensions = AssemblyHelper.ExtensionsForSystemDrawing();
-            return (extensions != null) ? extensions.IsBitmap(data) : false;
-        }
+        /// <inheritdoc cref="SystemDrawingExtensionMethods.IsBitmap(object?)"/>
+        internal static bool IsBitmap(object? data) =>
+            AssemblyHelper.ExtensionsForSystemDrawing()?.IsBitmap(data) ?? false;
```

Comment:
What's the benefit of these changes? Only makes it more difficult to debug to not have the `extensions` as explicit step and it gets optimized away anyway.

Reply:
More class context on the screen at the same time. The impact to debugging doesn't seem to outweigh that benefit for something this simple.